src/osd/scrubber/osd_scrub_sched.cc

```diff
@@ -800,3 +800,24 @@ void ScrubQueue::dump_scrub_reservations(ceph::Formatter* f) const
   f->dump_int("scrubs_remote", scrubs_remote);
   f->dump_int("osd_max_scrubs", conf()->osd_max_scrubs);
 }
+
+void ScrubQueue::clear_pg_scrub_blocked(spg_t blocked_pg)
+{
+  dout(5) << fmt::format(": pg {} is unblocked", __func__, blocked_pg)
+	  << dendl;
+  --blocked_scrubs_cnt;
+  ceph_assert(blocked_scrubs_cnt >= 0);
+}
+
+void ScrubQueue::mark_pg_scrub_blocked(spg_t blocked_pg)
+{
+  dout(5) << fmt::format(": pg {} is blocked on an object",
+			 blocked_pg)
+	  << dendl;
```

Comment:
Would we want `__func__` to be included in the log message here?

Reply:
dout() for ScrubQueue already includes some data items, including __func__